sql/catalyst/src/main/scala/org/apache/spark/sql/QueryCompilationErrors.scala

```diff
@@ -268,6 +268,11 @@ object QueryCompilationErrors {
       s"but $prettyName is not an aggregate function")
   }
 
+  def ignoreNullsWithUnsupportedFunctionError(prettyName: String): Throwable = {
+    new AnalysisException("IGNORE NULLS specified, " +
+      s"but $prettyName is not first, last or an offset window function")
```

Comment:
we can simply say `Function $prettyName does not support IGNORE NULLS`

Reply:
OK